console/db/migrations/20241205103951_2.1.0.sql

```diff
@@ -23,3 +23,14 @@ WHERE extension_name IN (
 -- +goose Down
 DELETE FROM public.postgres_versions 
 WHERE major_version = 17;
+
+
+-- Extends 20240520144338_2.0.0_initial_scheme_setup.sql#L217 with more cloud instance types
+-- The price is approximate because it is specified for one region and may differ in other regions.
+-- aws, gcp, azure: the price is for the region 'US East'
+INSERT INTO public.cloud_instances (cloud_provider, instance_group, instance_name, cpu, ram, price_hourly, price_monthly, currency, updated_at) VALUES
+    ('hetzner', 'Small Size', 'CX22', 2, 4, 0.0064 , 3.99, '$', '2024-12-10'),    
+    ('hetzner', 'Small Size', 'CX32', 4, 8, 0.0117 , 6.99, '$', '2024-12-10'),
+    ('hetzner', 'Medium Size', 'CX42', 8, 16, 0.0294 , 17.99, '$', '2024-12-10'),
+    ('hetzner', 'Medium Size', 'CX52', 16, 32, 0.0601 , 35.49, '$', '2024-12-10');
```

Comment:
Please place above '+goose Down' as the rollback scripts are described here.

Reply:
What about shared CPUs based on AMD? I believe they should also be added, as it seems, based on the Intel-based pricing page, that they are not available in the USA and Singapore, unlike the **CPX** types.

Reply:
Moved the `+goose Down` to the end of the file:  https://github.com/vitabaks/autobase/pull/835/commits/a73c8087709a2ab0258dabbf0df3fd1270987073